pages/national_resources/uk_resources.md

```diff
@@ -70,6 +70,7 @@ This page provides an overview of the data management resources in the UK. The t
    * [Wellcome Trust](https://wellcome.org/): [DM and sharing guidelines](https://wellcome.org/grant-funding/guidance/data-software-materials-management-and-sharing-policy)
    * [The Royal Society](https://royalsociety.org/): [DM and sharing guidelines](https://royalsociety.org/journals/ethics-policies/data-sharing-mining/)
    * [Cancer Research UK](https://www.cancerresearchuk.org/): [DM and sharing guidelines](https://www.cancerresearchuk.org/funding-for-researchers/applying-for-funding/policies-that-affect-your-grant/submission-of-a-data-sharing-and-preservation-strategy/data-sharing-guidelines)
+   * [British Heart Foundation](https://www.bhf.org.uk/): [DM and sharing guidelines](https://www.bhf.org.uk/for-professionals/information-for-researchers/what-we-fund)
```

Comment:
```suggestion
   * [British Heart Foundation](https://www.bhf.org.uk/): [DM and sharing guidelines](https://www.bhf.org.uk/for-professionals/information-for-researchers)
```

Reply:
@robertmand what do you think of this proposed change?